frontend/src/lib/k8s/index.test.ts

```diff
@@ -55,3 +55,148 @@ describe('Class tests', () => {
     });
   });
 });
+
+const labelSelector: LabelSelector = {
```

Comment:
I remember that @joaquimrocha  and @illume mentioned about test infra for Headlamp. Are these test cases prepared for that?

Reply:
Unfortunately testing the label selector effectively would have to involved the k8s API server (as mocking the results wouldn't make much sense in this case).
We maybe have a way to do this now that we just added e2e testing.

Reply:
I think this conversation can be resolved... but not sure. Can one of you please resolve it, or let me know what needs to be done/discussed?